config/default.json

```diff
@@ -1,3 +1,5 @@
 {
-  "mongoURI": "mongodb+srv://user:1234@group-f-l1h2e.mongodb.net/test?retryWrites=true&w=majority"
+  "mongoURI": "mongodb+srv://user:1234@group-f-l1h2e.mongodb.net/test?retryWrites=true&w=majority",
```

Comment:
Jason suggested to not have any password in github

Reply:
yes I put it back in because I was unsure if it would break the app if I gitignored the folder.

Reply:
got it, I hope my change request does not interfere the merging